### ISSUE TITLE
Which is validation set？

### ISSUE THREAD

**Issue by EmilyBuffy** [dykuang/Medical-image-registration#13]
Dear dykuang,
      Your work has inspired me a lot, but there is some code  I got confused.
      1. In ` train.py`,  `val_files =  ['{:03d}'.format(i) for i in range(0, 20)]` 
      Which data subset is it(val_files) in mindboggle? I guess it should not be OASIS-TRT-20, because this subset is used to evaluate the network as test set.  
      2. In addition. In voxelmorph, when Unet is down-sampling, the image size will be reduced by a factor of 1/2. If the data set is cropped to 144x180x144, when reproducing voxelmorph, the value 180 will produce fractions during down-sampling,which is automatically rounded down by the computer. Then the size of the image after down sampling will not match the up-sampling, wouldn't it make it impossible to train? 
Thanks

**Comment by dykuang** [comment]
1. I think there were 5 or 6 sub datasets in Mingboggle101? You can make a free split of your train/val/test split. Say 3/1/2 split. 
Or you can just mix them and take randomly from the 101 samples to form the train/val/test sets.  
 
2.  Yes, you are right. I think we added marginal padding in Voxelmorph to make the number even when dividing by 2 and crop the border when upsampling to make the size match. As a probably more convenient approach, you can simply resize the volume to resolutions with power of 2 at the very beginning.
